frontend/src/scenes/insights/InsightsNav.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button, Row, Tabs, Tooltip } from 'antd'
```

Comment:
As a small piece of feedback, it would help speed the review, if you add a few comments inline describing some of this refactoring changes (e.g. this was moved from Insights verbatim).

Reply:
I think it'd be more useful in the PR (as if you were reviewing your own code) as it'll be meaningless after the PR is merged.